dashboard/test/controllers/admin_search_controller_test.rb

```diff
@@ -135,17 +135,17 @@ class AdminSearchControllerTest < ActionController::TestCase
   test 'admin can create pilot' do
     post :create_pilot, params: {
       pilot: {
-        name: 'test-pilot-1',
-        display_name: 'Test Pilot 1',
+        name: 'test-pilot-2',
```

Comment:
fyi @jamescodeorg I was having Drone failures I couldn't repro locally. I switched this test to use `test-pilot-2` and that seemed to fix it but I don't know why it would've failed in the first place. If we see flaky tests in the future we might need to investigate this further